docs/clusters/how-to-upgrade-the-temporal-server-version.md

```diff
@@ -20,6 +20,10 @@ If a database schema upgrade is required, it will be called out directly in the
 Some releases require changes to the schema, and some do not.
 We ensure that any consecutive versions are compatible in terms of database schema upgrades, features, and system behavior, however there is no guarantee that there is compatibility between _any_ 2 non-consecutive versions.
 
+When upgrading your Temporal Server version, ensure that you upgrade sequentially. For example, when upgrading from v1.n.x, always upgrade to v1.n+1.x (or the next available version) and so on until you get to the required version.
```

Comment:
```suggestion
When upgrading your Temporal Server version, ensure that you upgrade sequentially.
For example, when upgrading from v1.[n].[x], always upgrade to v1.[n].[x+1] (or the next available version) and so on until you get to the required version.
```
The sense is that the user needs to upgrade from v1.18.1 to v1.18.2 to v1.18.3 up to v1.18.[max], right? And then increment _n_ if appropriate? (Some things are a lot easier to write in code than in a natural language.)